EIPS/eip-5553.md

```diff
@@ -0,0 +1,260 @@
+
+With this proposal, registration is taken care of, and splits are taken care of by defining the splits as an open array of EIP-20 tokens that represent split percentages. Moving splits to various participants will be as simple as sending an EIP-20 token from one wallet to another. discovering split movements is as simple as checking EIP-20 transfer events.
+
+In the future, we can build full payout mechanisms based on these simple split tokens, and licensing standards on top of those payout mechanisms, which are not possible today due to the complexity of splits management in the traditional web2 world.
+
+## Specification 
+
+The key words "MUST", "MUST NOT", "REQUIRED", "SHALL", "SHALL
+NOT", "SHOULD", "SHOULD NOT", "RECOMMENDED", "MAY", and
+"OPTIONAL" in this document are to be interpreted as described in
+RFC 2119.
+
+**contracts that want to represent a work on chain MUST implement [EIP-721](./eip-721.md) AND this Proposal**
+
+Implementers of this standard **MUST** have all of the following functions:
+
+```solidity
+// SPDX-License-Identifier: Apache-2.0
+pragma solidity ^0.8.9;
+import '@openzeppelin/contracts/interfaces/IERC165.sol';
+
+
+///
+/// @dev Interface for Simple Works Registration Standard
+///
+interface IWorksRegistration is IERC165 {
+    
+    /// @notice Called with the new uri to an updated metadata file
+    /// @param _newUri - the uri pointing to a metadata file (file standard is up to the implementer)
+    /// @param _newFileHash - The hash of the new metadata file, for future reference and verification
+    function changeMetadataURI(string memory _newUri, string memory _newFileHash) external ;
+
+    /// @return array of addresses of ERC20 tokens representing royalty interest in the work
+    /// @dev i.e implementing ERC5501 (IRoyaltyInterestToken interface)
+    function royaltyInterestTokens() external view returns (address[] memory) ;
+
+    /// @return the address of the contract or EOA that initialized the work registration
+    /// @dev i.e a registry or registrar, to be implemented in the future
+    function ledger() external view returns (address) ;
+
+    /// @return the uri of the current metadata file for the work
+    function metadataURI() external view returns (string memory) ;
+
+    /// @dev event to be triggered whenever metadata URI is changed
+    /// @param byAddress the addresses that triggered this operation
+    /// @param oldURI the uri to the old metadata file prior to the change
+    /// @param oldFileHash the hash of the old metadata file prior to the change
+    /// @param newURI the uri to the new metadata file 
+    /// @param newFileHash the hash of the new metadata file 
+    event MetadaDataChanged(address byAddress,string oldURI, string oldFileHash, string newURI, string newFileHash);
+}
+```
+
+### Examples 
```

Comment:
[Editorial] Move to "Reference Implementation" section?

Reply:
Renamed.